engines/dfc_provider/app/controllers/dfc_provider/application_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 # Controller used to provide the API products for the DFC application
 module DfcProvider
-  class BaseController < ActionController::Base
+  class ApplicationController < ActionController::Base
```

Comment:
Took me a while to understand what was happening here. The [rule](https://www.rubydoc.info/gems/rubocop-rails/RuboCop/Cop/Rails/ApplicationController) says:

> Checks that controllers subclass ‘ApplicationController`.

But I think what the rule should  _really_ say is:

_Checks that the application uses a class named `ApplicationController`, and it is the only class that directly inherits from `ActionController::Base`_